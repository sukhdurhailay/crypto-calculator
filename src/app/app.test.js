import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import App from './app.ui';

describe('App', () => {
  it('renders without crashing', () => {
    shallow(<App />);
  });

  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
