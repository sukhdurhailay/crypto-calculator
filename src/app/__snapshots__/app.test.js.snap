// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="sc-bdVaJa eGJdiE"
>
  <h1>
    CryptoCalc
  </h1>
</div>
`;
